src/adapter/etl-adapter-google-sheet/composer.json

```diff
@@ -28,7 +28,7 @@
             ]
         },
         "files": [
-            "src/Flow/ETL/Adapter/GoogleSheet/functions.php"
+            "src/Flow/ETL/Adapter/GoogleSheet/DSL/functions.php"
```

Comment:
this requires a note in documentation/upgrading.md

Reply:
So maybe revert?

Reply:
nah, it's better this way and this adapter is not yet popular enough so it's probably best time for such change, just put one line in upgrading.md and will be perfect

Reply:
Added.